velox/substrait/VeloxToSubstraitExpr.cpp

```diff
@@ -207,6 +229,42 @@ VeloxToSubstraitExprConvertor::toSubstraitLiteral(
   return *literalExpr;
 }
 
+void VeloxToSubstraitExprConvertor::complexVectorToLiteral(
+    google::protobuf::Arena& arena,
+    std::shared_ptr<ConstantVector<ComplexType>> constantVector,
+    ::substrait::Expression_Literal* substraitLiteral) {
+  VELOX_CHECK_EQ(
+      constantVector->size(), 1, "Only one complex vector is expected.");
```

Comment:
> Only one complex vector

This message is misleading. There is only vector, constantVector, but it may have multiple rows. Perhaps, change to "Constant vector is expected to have only one row.".